### ISSUE TITLE
"Heading" in "Group" results in error in email template preview

### ISSUE THREAD

**Issue by andreasottosson** [verbb/formie#656]
**Description**
Trying to view the preview for email notifications on a form with a field of type "Heading" placed inside a "Group" field results in the following error (stack trace from Craft's logs)

```
2021-11-25 11:03:31 [-][1][l3f3gbjk4am9olsqoo7fl8ehj3][error][yii\base\UnknownPropertyException] yii\base\UnknownPropertyException: Setting unknown property: verbb\formie\models\FakeElement::kontaktinformatoner in /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/base/Component.php:210
Stack trace:
#0 /home/somedomain.com/tsakansdzy/public_html/vendor/craftcms/cms/src/base/Element.php(1751): yii\base\Component->__set('kontaktinformat...', 'Placeat est qui...')
#1 /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/BaseYii.php(558): craft\base\Element->__set('kontaktinformat...', 'Placeat est qui...')
#2 /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/base/BaseObject.php(107): yii\BaseYii::configure(Object(verbb\formie\models\FakeElement), Array)
#3 /home/somedomain.com/tsakansdzy/public_html/vendor/verbb/formie/src/models/FakeElementQuery.php(22): yii\base\BaseObject->__construct(Array)
#4 /home/somedomain.com/tsakansdzy/public_html/vendor/verbb/formie/src/services/Submissions.php(671): verbb\formie\models\FakeElementQuery->setFieldValues(Array, Object(verbb\formie\models\FieldLayout))
#5 /home/somedomain.com/tsakansdzy/public_html/vendor/verbb/formie/src/services/Submissions.php(555): verbb\formie\services\Submissions->_getFakeFieldContent(Array)
#6 /home/somedomain.com/tsakansdzy/public_html/vendor/verbb/formie/src/controllers/EmailController.php(114): verbb\formie\services\Submissions->populateFakeSubmission(Object(verbb\formie\elements\Submission))
#7 /home/somedomain.com/tsakansdzy/public_html/vendor/verbb/formie/src/controllers/EmailController.php(27): verbb\formie\controllers\EmailController->_populateFromPost(Object(verbb\formie\models\Notification), Object(verbb\formie\elements\Submission))
#8 [internal function]: verbb\formie\controllers\EmailController->actionPreview()
#9 /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#10 /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#11 /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction('preview', Array)
#12 /home/somedomain.com/tsakansdzy/public_html/vendor/craftcms/cms/src/web/Application.php(287): yii\base\Module->runAction('formie/email/pr...', Array)
#13 /home/somedomain.com/tsakansdzy/public_html/vendor/craftcms/cms/src/web/Application.php(596): craft\web\Application->runAction('formie/email/pr...', Array)
#14 /home/somedomain.com/tsakansdzy/public_html/vendor/craftcms/cms/src/web/Application.php(266): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#15 /home/somedomain.com/tsakansdzy/public_html/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#16 /home/somedomain.com/tsakansdzy/public_html/web/index.php(26): yii\base\Application->run()
#17 {main}
2021-11-25 11:03:31 [-][1][l3f3gbjk4am9olsqoo7fl8ehj3][info][application] $_GET = [
    'p' => 'admin/actions/formie/email/preview'
]
```

**Steps to reproduce**
1. Add a field of type "Heading" to a "Group"
2. Try to view email template preview

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No
- Custom Email Notifications Template: Yes

**Additional info**
- Plugin version: 1.4.26
- Craft version: 3.7.21
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.26",
  "...": "..."
}
```

Then run `composer update`.

**Comment by andreasottosson** [comment]
Awesome, thanks! @engram-design

**Comment by engram-design** [comment]
Fixed in [1.4.27](https://github.com/verbb/formie/releases/tag/1.4.27)
